docs/modules/callbacks/how_to_guides.rst

```diff
@@ -0,0 +1,28 @@
+How-To Guides
+=============
+
+LangChain provides a callbacks system that allows you to hook into the various stages of your LLM application. 
+This is useful for logging, `monitoring <https://python.langchain.com/en/latest/tracing.html>`_, 
```

Comment:
The same. This (and others, below) link should be relative, like ./callbacks/how_to_guides.html, no?

Reply:
Makes a lot of sense! I just used those as I saw it on another `how_to_guides.rst` file, but maybe it's worth double-checking and replacing those URLs if needed 👍🏻